fusillade-api.yml

```diff
@@ -26,7 +26,7 @@ info:
 
   version: $FUS_VERSION
 servers:
-  - url: https://$API_DOMAIN_NAME
+  - url: https://$API_DOMAIN_NAME/v1
 paths:
   /login:
```

Comment:
Will this change the login URL from `$API_DOMAIN_NAME/login` to `$API_DOMAIN_NAME/v1/login`? Is that something we're okay with?

If not, we can add a `server` directive beneath the endpoints we don't want prefixed with `/v1`

Reply:
I think this might be the opposite of (or similar to) what @Bento007 was suggesting in [this comment](https://github.com/HumanCellAtlas/fusillade/pull/295#issuecomment-546490457)

Reply:
So i've tested this out. While openAPI says it's valid connexion does not use it. https://connexion.readthedocs.io/en/latest/routing.html#api-versioning-and-basepath

Reply:
We could fork connexion and add it or deal with break the api with v1/login ect...

Reply:
yeah it seems that performing a server-override would be the best option for these paths, although i keep running into
```
connexion.exceptions.InvalidSpecification: 'server' does not match any of the regexes: '^x-'

Failed validating 'additionalProperties' in schema['properties']['paths']['patternProperties']['^/']['properties']['get']:
    {'additionalProperties': False,
     'description': 'Describes a single API operation on a path.',
     'patternProperties': {'^x-': {'$ref': '#/definitions/specificationExtension'}},
     'properties': {'callbacks': {'$ref': '#/definitions/callbacksOrReferences'},
                    'deprecated': {'type': 'boolean'},
                    'description': {'type': 'string'},
                    'externalDocs': {'$ref': '#/definitions/externalDocs'},
                    'operationId': {'type': 'string'},
                    'parameters': {'items': {'$ref': '#/definitions/parameterOrReference'},
                                   'type': 'array',
                                   'uniqueItems': True},
                    'requestBody': {'$ref': '#/definitions/requestBodyOrReference'},
                    'responses': {'$ref': '#/definitions/responses'},
                    'security': {'items': {'$ref': '#/definitions/securityRequirement'},
                                 'type': 'array',
                                 'uniqueItems': True},
                    'servers': {'items': {'$ref': '#/definitions/server'},
                                'type': 'array',
                                'uniqueItems': True},
                    'summary': {'type': 'string'},
                    'tags': {'items': {'type': 'string'},
                             'type': 'array',
                             'uniqueItems': True}},
     'required': ['responses'],
     'type': 'object'}

On instance['paths']['/login']['get']:
    {'description': 'Send the user agent to an identity provider selector '
                    "and generate a user account to establish the user's "
                    'identity. This is a redirect endpoint.',
     'operationId': 'fusillade.api.oauth.login',
     'parameters': [{'description': 'Where to redirect to once login is '
                                    'complete.',
                     'in': 'query',
                     'name': 'redirect_uri',
                     'required': True,
                     'schema': {'type': 'string'}},
                    {'description': 'An opaque parameter that is returned '
                                    'back to the `redirect_uri`.',
                     'in': 'query',
                     'name': 'state',
                     'schema': {'type': 'string'}}],
     'responses': {'301': {'description': 'OK.'},
                   'default': {'description': 'Unexpected error.'}},
     'server': [{'description': 'Override base path for /login operations',
                 'url': 'https://auth.dev.data.humancellatlas.org'}],
     'summary': 'Establish the users identity using the OIDC provider',
     'tags': ['oauth']}

----------------------------------------------------------------------
```

perhaps its how im forming the yml.

Reply:
the issue above was resolved, I think I'm also stuck in the same spot that @Bento00 was in, which is interfacing with connexion, i see that this issue was already raised within connexion, https://github.com/zalando/connexion/issues/1047